### PR TITLE
Update deployment process for slotless deployment

### PR DESCRIPTION
**Pull request by tomnatt**
Removing the badger deployment documentation as we've now properly switched over and tidied up the page.

### CODE REVIEW

File: source/manual/deploying.html.md [hunk 1]
```diff
@@ -27,12 +27,13 @@ We have a staging environment that must always be deployed to immediately before
...
-multiple teams are involved in the release, pick one.
+multiple teams are involved in the release, pick one. Releases are tracked by the [release app](https://release.publishing.service.gov.uk/).
 
+You need to have a Signon account with appropriate permissions to access the release app.
 
-## Experimental deployment process
+## Deployment process
 
-As a response to [RFC-70](https://gov-uk.atlassian.net/wiki/pages/viewpage.action?pageId=130703581) starting 15 May 2017 we are experimenting with a process which allows us to remove the need to book a deployment slot. If you think there's something wrong with the process during the experiment speak up as soon as possible.
+As a response to [RFC-70](https://gov-uk.atlassian.net/wiki/pages/viewpage.action?pageId=130703581) starting 15 May 2017 we are using a process which allows us to deploy without the need for a booked deployment slot.
```
Comment (timblair) [inline source/manual/deploying.html.md]: The link to the RFC should point to the [RFC on GitHub](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-070-path-towards-continuous-deployment-cd.md) now.

Comment (tomnatt) [inline source/manual/deploying.html.md]: Yup, updated.

File: source/manual/deploying.html.md [hunk 2]
```diff
@@ -75,72 +80,6 @@ trying again later.
...
-
-### Deployment
-
-#### Security fixes
-
-If you are responding to a security incident, follow the steps in [Deploy fixes for a security vulnerability](deploy-fixes-for-a-security-vulnerability.html).
-
-#### Regular code changes
-
-1.  Check any notes against the application in the [Release app][release].
```
Comment (chao-xian) [inline source/manual/deploying.html.md]: Most of this list (the bits about checking stuff) still looks useful to me.

Comment (tomnatt) [inline source/manual/deploying.html.md]: The non-badger stuff is duplicated in the first "deployment" section.

**Pull request merged**